.github/ISSUE_TEMPLATE/security-vulnerabilities-report.yml

```diff
@@ -1,34 +1,94 @@
   attributes:
-    label: 'Other security advisory IDs'
-    description: 'Other security advisory IDs. Please list with references to each ID.'
+    label: 'CVE IDs'
+    description: 'Comma-separated list of CVE IDs. Fill N/A if no CVE ID is assigned.'
+    placeholder: 'CVE-20xx-xxxx[, CVE-20xx-xxxx, ...]'
   validations:
     required: true
+# Severity is a new field of the template. This may help maintainers to identify which issue is most urgent and require attentions.
+- type: dropdown
+  attributes:
+    label: 'Severity'
+    description: | 
+      Select the severity of reporting vulnerabilities. Refer to our wiki for details.
+      Choose the highest in case of multiple vulnerabilities.
+    #Through there is no such wiki yet.
+    options:
+      - 'Critical'
+      - 'High'
+      - 'Moderate'
+      - 'Low'
 - type: textarea
   attributes:
-    label: 'Description'
-    description: 'Original mailing list message(s) or other description here.'
+    label: 'Other security advisories'
+    description: 'Other security advisories, mailing list archive or original bug report. Please list with references to each ID.'
+    placeholder: |
+      Examples:
+      - Debian: [DSA-xxxx-x](link_to_DSA);
+      - Links to original mail list archive;
+      - Links to original issue, if applicable
   validations:
     required: true
 - type: textarea
   attributes:
-    label: 'Patches'
-    description: 'Enter "N/A" if version update is preferred, otherwise please label each patch with ref. to original vulnerability ID(s).'
+    label: 'Description'
+    description: |
+      Original mailing list message(s) or other description may list here. 
+      You may also include these following supplemental details:
+      - Bug introduced in which version
+      - Possible security impact on AOSC OS
+      - Available PoCs (Required if the reporting target is AOSC-developed component)
+      - Available patches or fixed version for this specific CVE(s)
+      - How to fix that problem
+      - Linked CWE if available. 
+      - ...
+    placeholder: |
+      Examples (for GNU tar):
+      - CVE-2022-48303: 
+        - (Upstream description pastes here)
+        - Impact: May only have local impact such as crashing. Attack requires a specific malformed tar file
+        - Severity is escalated as a component of basic OS experience.
+        - Available PoC: https://savannah.gnu.org/bugs/?62387
+        - Patch: https://savannah.gnu.org/patch/?10307
+        - Fix: Patch only
+        - CWE: CWE-125 Out-of-bound read
+      - (Other vulnerabilities) ...
   validations:
     required: true
+# The original `patches` is renamed to `Fixed version or patches`. If an package update is suggested the reporter should include the latest version and the earliest fixed version if applicable. 
 - type: textarea
   attributes:
-    label: 'PoC(s)'
-    description: 'Please list links to available PoCs (Proofs of Concept).'
+    label: 'Fixed versions or patches'
```

Comment:
This was already done with title + label, this is not necessary.

Reply:
May be called `solutions`? This field is intended to replace original `Patches`.